src/DependencyInjection/CompilerPass/LiveTwigComponentCompilerPass.php

```diff
@@ -26,5 +28,19 @@ public function process(ContainerBuilder $container): void
             ->addTag('sylius.live_component.admin', $tagConfig[0])
             ->clearTag('sylius.twig_component')
         ;
+
+        $summaryComponentDefinition = new Definition(SummaryPaymentComponent::class);
+        $summaryComponentDefinition
+            ->setArguments([
+                new Reference('request_stack'),
+                new Reference('sylius.repository.payment_request'),
+                new Reference('monolog.logger'),
+            ])
+            ->addTag('twig.component', [
+                'key' => 'sylius_shop:order_pay:web_elements:summary',
+                'template' => '@SyliusShop/cart/index/content/form/sections/general/summary.html.twig'
+            ]);
+
+        $container->setDefinition('sylius_live_component.payment_summary', $summaryComponentDefinition);
     }
```

Comment:
This definition can be added to the `config/services/integrations/sylius_shop/live_component.yaml`.
But like I wrote earlier, this live component can be useless if the templates are simply displaying info from the `PaymentIntent` object.

A `CompilerPass` is required to modify something available in another bundle (here it allows the Sylius PaymentMethod admin form to be a LiveComponent).